src/System.IO.Pipelines/tests/PipeReaderWriterFacts.cs

```diff
@@ -662,5 +766,24 @@ public void GetMemoryZeroReturnsNonEmpty()
         {
             Assert.True(_pipe.Writer.GetMemory(0).Length > 0);
         }
+
+        private sealed class CustomSynchronizationContext : SynchronizationContext
+        {
+            public override void Post(SendOrPostCallback d, object state)
```

Comment:
Don't use single letter arg names.

Reply:
Funnily enough, I copied this from https://github.com/dotnet/corefx/blob/3fca26cedf6c54aabc100e69106fc7e263db7815/src/System.Net.Sockets/tests/FunctionalTests/NetworkStreamTest.netcoreapp.cs#L352. 

Also this is the actual arg name in the base class https://msdn.microsoft.com/en-us/library/system.threading.synchronizationcontext.send(v=vs.110).aspx.

Given that I feel the need to leave it alone.